libportal/portal-qt5.h

```diff
@@ -21,7 +21,85 @@
 
 #include <libportal/portal.h>
 
+#include <QMap>
+#include <QStringList>
+#include <QSharedPointer>
+#include <QVariant>
 #include <QWindow>
 
 XDP_PUBLIC
 XdpParent *xdp_parent_new_qt (QWindow *window);
+
+// Account portal helpers
+typedef struct {
```

Comment:
This is already C++, so we don't need to use typedefs for declaring structs, no?

```
struct XdpQtGetUserInformationResult {
    QString id;
    QString name;
    QString image;
};
```